datafusion/src/physical_plan/metrics/value.rs

```diff
@@ -401,10 +413,12 @@ impl MetricValue {
         match self {
             Self::OutputRows(_) => 0,     // show first
             Self::ElapsedCompute(_) => 1, // show second
-            Self::Count { .. } => 2,
-            Self::Time { .. } => 3,
-            Self::StartTimestamp(_) => 4, // show timestamps last
-            Self::EndTimestamp(_) => 5,
+            Self::SpillCount(_) => 2,
+            Self::SpilledBytes(_) => 3,
+            Self::Count { .. } => 4,
```

Comment:
It looks good to me, but we can make the format consistent with others as a minor fix in a follow-up pull request.
`Self::Count { .. } => 4,` -> `Self::Count(_) => 4,`